src/includes/class-health-check-wp-debug.php

```diff
@@ -0,0 +1,536 @@
+<?php
+
+/**
+ * Enables / disables WP_DEBUG.
+ *
+ * @package Health Check
+ */
+
+/**
+ * Class Health Check WP Debug
+ */
+class Health_Check_WP_Debug {
+
+	/**
+	 * Enables WP_DEBUG
+	 *
+	 * @uses copy()
+	 * @uses file()
+	 * @uses file_put_contents()
+	 * @uses fopen()
+	 * @uses strpos()
+	 * @uses fputs()
+	 * @uses fclose()
+	 * @uses wp_send_json_error()
+	 * @uses wp_send_json_succes()
+	 *
+	 * @return void
+	 */
+	static function enable_wp_debug() {
```

Comment:
When enabling debugging we need to do a loopback test, to ensure our config changes didn't break the users site. If it did break we need to restore the config backup. If the loopback is successful, we can delete the backup file. We need to do the same when disabling debugging and inform the user that debugging may still be active as the changes encountered an error of some kind.